### PR TITLE
[chore] move awsemf exporter to generated lifecycle tests

### PR DESCRIPTION
**Pull request by Frapschen**
Relates to https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27849

### CODE REVIEW

File: exporter/awsemfexporter/metadata.yaml [hunk 1]
```diff
@@ -7,3 +7,10 @@ status:
   distributions: [contrib, aws, observiq]
   codeowners:
     active: [Aneurysm9, shaochengwang, mxiamxia, bryan-aguilar]
+
+tests:
+  config:
+    region: 'us-west-2'
+    resource_to_telemetry_conversion:
+      enabled: true
```
Comment (bryan-aguilar) [inline exporter/awsemfexporter/metadata.yaml:15]: Are these required for the tests to pass?

Comment (atoulme) [reply exporter/awsemfexporter/metadata.yaml:15]: I checked - yes, tests fail otherwise.

**Comment by atoulme** [comment]
Pinging @Aneurysm9 @shaochengwang @mxiamxia and @bryan-aguilar for codeowner review.

File: exporter/awsemfexporter/metadata.yaml (hunk 1 above)
Comment (bryan-aguilar) [reply exporter/awsemfexporter/metadata.yaml:15]: Interesting.

**Review by bryan-aguilar — APPROVED** [review]

**Review by atoulme — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**